packages/api/src/routers/api/alerts.ts

```diff
@@ -71,6 +72,52 @@ const validateGroupBy = async (req, res, next) => {
 };
 
 // Routes
+router.get('/', isUserAuthenticated, async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    if (teamId == null) {
+      return res.sendStatus(403);
+    }
+    const alerts = await Alert.find({ team: teamId });
```

Comment:
I wonder if we want to populate associated dashboard or chart record for the chart type alerts. It really depends on what we show on the frontend side